bulletin_board/server/app/models/election.rb

```diff
@@ -8,7 +8,7 @@ class Election < ApplicationRecord
 
   has_one_attached :verifiable_results
 
-  enum status: { created: 0, key_ceremony: 1, key_ceremony_ended: 2, vote: 3, vote_ended: 4, tally: 5, tally_ended: 6, results_published: 7 }
+  enum status: { created: 0, key_ceremony: 1, key_ceremony_ended: 2, vote: 3, vote_ended: 4, tally_started: 5, tally_ended: 6, results_published: 7 }
```

Comment:
Just FYI, this change needed to be done because Ruby 2.7 added `Enumerable#tally` which conflicts with this name:
https://ruby-doc.org/core-3.0.2/Enumerable.html#method-i-tally

There is an error printed in the console saying that the Active Record relation already has a method named `tally` which is why it cannot be used as an enumeration key.